tools/dockerfile/test/bazel/Dockerfile

```diff
@@ -25,6 +25,7 @@ RUN apt-get update && apt-get -y install \
   make \
   openjdk-8-jdk \
   vim
+RUN apt-get install --reinstall python-pkg-resources
```

Comment:
Could you add a comment explaining the reinstall? The reasoning you gave in your pull request description would be good.